py-polars/tests/test_udfs.py

```diff
@@ -113,6 +113,13 @@
     ("c", lambda x: json.loads(x), 'pl.col("c").str.json_extract()'),
 ]
 
+NOOP_TEST_CASES = [
```

Comment:
moving these here from `test_inefficient_apply`, we should probably be testing the "no-op" cases across all supported python versions too